image/image.go

```diff
@@ -412,7 +412,7 @@ func neededDefaultProviders(info *snap.Info) (cps []string) {
 // snaps. If not an error is returned.
 func hasBase(snap *snap.Info, local *localInfos, snaps []string) error {
 	// snap needs no base: nothing to do
-	if snap.Base == "" {
+	if snap.Base == "" || snap.Base == "none" {
```

Comment:
Is `snap.Base` being empty the same as `snap.Base` being equal to `none`? Is using `core` always explicit?

Reply:
no,  we never see "core",  core becomes empty (so there is some ambiguity left)

Reply:
@stolowski I think this warrants a comment.

Reply:
Added.